### ISSUE TITLE
Sprint 1 - Intake (Thomas)

### ISSUE THREAD

**Issue by Aletrioli** [sta-titansrobotics/27294-IntoTheDeep#14]
Goal for the week: Complete and test the starter robot intake design, learning about what an intake requires and how it is meant to function.

**Comment by Aletrioli** [comment]
October 8th:
I initially put some flap wheels on a core hex motor to see how rotating flap wheels would pull and push a block. I then began to follow the instructions of the starter robot intake design, placing flap wheels, a gear, collars, and pillows on a hex-shaft.

Video of initial test --> https://github.com/user-attachments/assets/d9b3fb64-8b5d-495f-ae0f-d72ef203a8fe
![IMG_7647](https://github.com/user-attachments/assets/de42a0ce-ca74-46ef-82a8-49096a51bb86)

**Comment by Aletrioli** [comment]
October 9th:
I continued to work on the intake, adding an extension frame to it, meant for slotting on the servo and attaching to the arms of our robot. I intended to attach the servo on this day, but accidentally used a metal frame instead of a plastic frame, and did not have enough time to correct my issue before class ended.

Obstacles:
- The empty space on the hex shaft gives the flap wheels some space to spread out which could result in inconsistency

Video of spinning flaps --> https://github.com/user-attachments/assets/36847257-f7b9-4606-b9c2-01f8402bf8ec
![unnamed](https://github.com/user-attachments/assets/9669c54d-d81f-47e0-aae3-a01fa5204e93)

**Comment by Aletrioli** [comment]
October 10th:
I attached the servo, linked up the gears, and added the ramp to the intake. On this day the starter build was "finished".

Obstacles:
- The plastic was cut and attached imprecisely, meaning that at some points of the ramp it is far more difficult to pick up samples than others
- The ramp's angle, while not steep, would benefit from being shallower, as the samples have to be pulled up the slope, rather than simply scooped onto a flat surface

![unnamed](https://github.com/user-attachments/assets/fd4249b9-3a9e-4dc7-9709-b72e6a04157d)
![unnamed](https://github.com/user-attachments/assets/68ff9dae-83ab-4aec-a4e0-7da36e4e7d48)

**Comment by Aletrioli** [comment]
October 11th:
To start with, I added an extrusion to the middle in order to create a space where I could attach the intake to the slider created by Kailyn and Hasini. We tested how the slider would hold up under the weight of the intake when fully extended, and found that it was wobbly. As a result, we now know that we must either lighten the intake or add a second slider. NEar the end of the day, I tried to add some wheels to the intake to allow it to roll along the floor.

Obstacles:
- Wheels were attached wrongly, resulting in the extrusion touching the floor
- Upon being attached to the intake, the slider was very unbalanced due to the weight of the intake being held at one point

![unnamed](https://github.com/user-attachments/assets/bbadecf4-7778-4d2f-bcd0-904bbaba9e55)
![CD21E6A4-D3CA-40CE-80F3-A6FB62B33A02](https://github.com/user-attachments/assets/c5e8d7df-1617-481a-a3b6-adf7d1d3eea1)
